forms-html/evan-hanan/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <form action="#">
+        <label for="email">Enter your email</label> <br>
+        <input type="email" name="email" id="email" required placeholder="Enter email"> <br>
```

Comment:
`placeholder` is usually used to give an example of a valid input. So here, you can do something like `jdoe@example.com` instead of `Enter email`. You can see this as an example: https://www.w3schools.com/tags/att_input_placeholder.asp